libraries/eddystone-advertising-library/README.md

```diff
@@ -0,0 +1,240 @@
+# Eddystone Advertising Library
+
+The Eddystone Advertising Library abstracts low level concepts of the Eddystone
+protocol and wraps existing Advertising APIs so that developers don't have to
+worry about these low level details. The library exposes simple functions that
+developers can use to advertise a Valid Eddystone packet from their device.
+
+** NOTE ** Currently only ChromeOS is supported.
+
+## Usage
+The Eddystone Advertising Library creates `window.eddystone` of type
+[`Eddystone`](#eddystone)
+
+To advertise a url:
+**Example**
+```js
+eddystone.registerAdvertisement({
+  type: 'url',
+  url: 'https://example.com',
+  txPower: -20
+}).then(advertisement => console.log("Advertising: " + advertisement.url))
+  .catch(error => console.log(error.message));
+```
+
+To stop advertising:
+**Example**
+```js
+advertisement.unregisterAdvertisement().then(() => {
+  console.log("Advertisment unregistered successfully.");
+}).catch(error => console.log(error.message));
+```
+Or if you have multiple advertisments:
+```js
+eddystone.advertisements.forEach(advertisement => {
+  advertisement.unregisterAdvertisement()
+    .then(() => console.log('Unregistered successfully'))
+    .catch(error => console.log('Couldn\'t unregister the advertisement: ' + error.message));
```

Comment:
maybe let's use double quotes like you do elsewhere to aid readability. (either that or tighten up the consistency in the files about single vs double)